.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,36 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Issue related to manifests
+    url: https://github.com/kubeflow/manifests/issues/new
```

Comment:
I think we can use `issues/new/choose` for all repos, because it auto redirects to `issues/new` if no templates are setup. When repos decide to set up some templates, they'll get what they want immediately.

Reply:
good idea